app/resources/documentation/manual/index.md

```diff
@@ -626,7 +626,7 @@ Both snap modes are used simultaneously. There may be situations when you have t
 
 The brush resize tool offers a quick way to move an individual face of a brush in 2D views. Hold #key(Alt) in addition to #key(Shift) when starting to drag a face in a 2D view to enable this mode. You will notice that a face is highlighted as usual, but when you start dragging the mouse, the face will just be moved in the direction you are dragging. The move is not restricted by the face normal, and other faces will be affected as well.
 
-![Moving faces](images/ResizeTool2DFaceMoving.gif)
+![Resizing multiple brushes](images/ResizeTool2DFaceMoving.gif)
```

Comment:
Wrong caption, the old one was correct

Reply:
Oh man, these changes were supposed to be in another PR that's been merged already. Apparently I forgot to save this file.

Regardless, I'll keep them in here even though it's sloppy. I have fixed this particular subtitle though.